test/lit/mlirCodegen/EraVM/bytecode-gen.sol

```diff
@@ -0,0 +1,8 @@
+// RUN: solc --mlir-action=gen-obj --mlir-target=eravm %s | FileCheck %s
+
+contract C {
+  function ret_42() public returns (uint256) { return 42; }
+}
+
+// NOTE: Assertions have been autogenerated by test/updFileCheckTest.py
+// CHECK: 0x00210000000000020000008001000039000000400010043f0000000001000416000000000101004b0000000d0000613d000000070000013d000000020300003900000000010000190000000002000019009500790000040f009500010000040f0000000d0000013d000000200100003900000100001004430000012000000443000001000100003900000040020000390000000203000039009500870000040f009500010000040f0000008001000039000000400010043f0000001f01000031000000040110008c000000370000413d0000001b0000013d00000002010003670000000001100370000000000101043b000000e001100270000000260110009c000000350000c13d000000220000013d0000000001000416000000000101004b0000002c0000613d000000260000013d000000000100001900000000020000190000000003000019009500790000040f009500010000040f0000002c0000013d009500770000040f0000000002010019000000400100043d000000000021043500000020020000390000000003000019009500870000040f009500010000040f000000360000013d000000360000013d000000370000013d000000000100001900000000020000190000000003000019009500790000040f009500010000040f00020000000000020001000000010353002000000000003f001f00000000003f001e00000000003f001d00000000003f001c00000000003f001b00000000003f001a00000000003f001900000000003f001800000000003f001700000000003f001600000000003f001500000000003f001400000000003f001300000000003f0002000000010355000000020d000031000000600dd00270001f002700d0019d0000001f0d00003100020000000d001d0001000000d103550000000000d103550003000000d103550004000000d103550005000000d103550006000000d103550007000000d103550008000000d103550009000000d10355000a000000d10355000b000000d10355000c000000d10355000d000000d10355000e000000d10355000f000000d103550010000000d103550011000000d103550012000000d10355001d00000002001f001300000003001f001400000004001f001500000005001f001600000006001f001700000007001f001800000008001f001900000009001f001a0000000a001f001b0000000b001f001c0000000c001f000000010120018f000000000101004b000000750000613d000000730000013d009500010000040f000000770000013d009500150000040f000000770000013d0000002a01000039000000000001042d0001000000000002000100000003001d00000000030200190000000102000029000000270410009c0000002701008041000000270430009c000000270300804100000040011002100000006003300210000000e002200210000000000131019f000000000121019f00000097000104300001000000000002000100000003001d00000000030200190000000102000029000000270410009c0000002701008041000000270430009c000000270300804100000040011002100000006003300210000000e002200210000000000131019f000000000121019f000000960001042e0000009500000432000000960001042e0000009700010430000000000000000000000000000000000000000000000000000000006808fd0700000000000000000000000000000000000000000000000000000000ffffffff0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
Is it a strict check?
Will break after merging an optimization.

Reply:
I added this just for the draft PR. I don't intend to keep it in the main branch.

Reply:
This output matches that of zksolc's `--bin` output. For now, would this work for the compiler-tester integration? Or should I wrap it in json?

Reply:
compiler-tester needs full standard JSON output like that of vanilla solc.
Replacing the EVM bytecode would be enough for now.